src/backend-c/runtime/constants.h

```diff
@@ -0,0 +1,715 @@
+#define TYPE_TUPLE_TAG 5
```

Comment:
We need to make sure that generating this file is idempotent. IE running/compiling the compiler multiple times never generates a larger file than compiling it once.

Reply:
Also if this file is generated by compiling the compiler then perhaps we shouldn't add this file to git.

Reply:
Done.